application/src/main/java/net/corda/v5/application/flows/FlowContextProperties.java

```diff
@@ -0,0 +1,52 @@
+package net.corda.v5.application.flows;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Context properties of a {@link Flow} are key value pairs of Strings. They are comprised of two sets of properties, those
+ * set by the platform when a {@link Flow} is started, and those which are added by the CorDapp developer during the execution
+ * of their flow. The latter set are referred to as user context properties. All context properties are immutable once
+ * set.
+ */
+public interface FlowContextProperties {
+    /**
+     * A constant containing the Corda reserved prefix.
+     */
+    String CORDA_RESERVED_PREFIX = "corda."; // constant must be lowercase
```

Comment:
Should this be `public static final`?

Reply:
Original was:

```kotlin
companion object {
        /**
         * A constant containing the Corda reserved prefix.
         */
        @JvmField
        val CORDA_RESERVED_PREFIX = "corda." // constant must be lowercase
    }
```

Reply:
`FlowContextProperties` is an `interface`, so `public static final` is implicit.

Reply:
👍